--- a/dsdt.dsl	2020-09-13 09:08:50.896207251 -0400
+++ b/dsdt.dsl	2020-09-21 14:28:52.428740000 -0400
@@ -1524,7 +1522,7 @@
         WOL7,   8
     }
 
-    Field (OGNS, DWordAcc, NoLock, Preserve)
+    Field (OGNS, AnyAcc, NoLock, Preserve)
     {
         Offset (0x1000), 
         ECON,   8, 
@@ -14112,14 +14110,7 @@
         {
             Method (_STA, 0, NotSerialized)  // _STA: Status
             {
-                If ((CIOE == One))
-                {
-                    Return (0x0F)
-                }
-                Else
-                {
-                    Return (Zero)
-                }
+                Return (0x0F)
             }
 
             Name (_HID, "INT343E")  // _HID: Hardware ID
@@ -38907,14 +38898,7 @@
 
             Method (_STA, 0, NotSerialized)  // _STA: Status
             {
-                If ((SCSS == One))
-                {
-                    Return (0x0F)
-                }
-                Else
-                {
-                    Return (Zero)
-                }
+                Return (0x0F)
             }
 
             Method (SSDB, 0, NotSerialized)
@@ -39961,4 +39945,3 @@
     {
     }
 }
-
