--- a/dsdt.dsl	2020-07-24 16:59:46.000000000 -0400
+++ b/dsdt.dsl	2020-09-23 11:29:20.587412067 -0400
@@ -1609,7 +1609,7 @@
         WOL7,   8
     }
 
-    Field (OGNS, DWordAcc, NoLock, Preserve)
+    Field (OGNS, AnyAcc, NoLock, Preserve)
     {
         Offset (0x1000), 
         ECON,   8, 
@@ -44760,14 +44760,9 @@
 
             Method (_STA, 0, NotSerialized)  // _STA: Status
             {
-                If ((SCSS == One))
-                {
-                    Return (0x0F)
-                }
-                Else
-                {
-                    Return (Zero)
-                }
+
+                Return (0x0F)
+
             }
 
             Method (SSDB, 0, NotSerialized)
